app/main/views/users.py

```diff
@@ -36,6 +37,15 @@ def get_user_by_id(user_id):
     return jsonify(users=user.serialize())
 
 
+@main.route('/users', methods=['GET'])
+def get_user_by_email():
+    email_address = request.args.get('email')
```

Comment:
Done - although I've gone with 404 rather than 400 due to arguments here (tl;dr: 400 is for syntax errors, and there's nothing wrong with the syntax in this case): http://stackoverflow.com/questions/9454811/which-http-status-code-to-use-for-required-parameters-not-provided

I'm not completely sold on 404 though so happy to go with 400 if you prefer.